values.yaml

```diff
@@ -252,9 +252,8 @@ server:
   connect: true
 
   # Resource requests, limits, etc. for the server cluster placement. This
-  # should map directly to the value of the resources field for a PodSpec,
-  # formatted as a multi-line string. By default no direct resource request
-  # is made.
+  # should map directly to the value of the resources field for a PodSpec.
+  # By default no direct resource request is made.
```

Comment:
Could we add a deprecation notice here and other values as well?

Reply:
Will do.